src/utils/transpile/index.js

```diff
@@ -6,15 +6,12 @@ export const generateElement = (
   { code = '', scope = {} },
   errorCallback
 ) => {
-  // NOTE: Workaround for classes, since buble doesn't allow `return` without a function
-  const transformed = transform(code)
-    .trim()
-    .replace(/^var \w+ =/, '')
-    .replace(/;$/, '')
+  // NOTE: Workaround for classes and arrow functions.
+  const transformed = transform(`(${code})`).trim()
```

Comment:
Bublé changed how standalone arrow functions were transpiled in https://github.com/Rich-Harris/buble/commit/f2bed343d68232f3dc77aab09555173f64acecb3 in such a way that we can't just take the inline result any more.

This workaround seems to suffice for both standalone arrow functions and for classes. It does add an IIFE for the class case, but I think that's probably okay.